login.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/3085842107.js" crossorigin="anonymous"></script>
+    <script defer src="/node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
+    <link rel="stylesheet" href="/dist/css/main.css">
+    <title>css frameworks</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div class="container">
+                <div class="row justify-content-center">
+                    <div class="col">
+                        <img src="assets/images/logo.png" alt="image of the phillers logo">
+                    </div>
+                    <hr>
+                    <div class="row mb-4">
+                      <div class="col-12">
+                        <nav class="nav align-items-center">
+                            <ul class="list-inline col-12 col-md-8 align-self-start d-flex justify-content-start">
```

Comment:
The parent of a col should be a row. It's probably best to separate out a new div here:

```suggestion
<div class="col-12 col-md-8">
  <ul class="list-inline row align-self-start justify-content-start">
```

Reply:
I'll add that when I get back on my computer. My workplace doesn't really give me opportunity to edit codes :D